docs/sources/search/_index.md

```diff
@@ -13,28 +14,38 @@ title: Search
 weight: 80
 ---
 
-# Search dashboards
+# Search dashboards and folders
 
-You can search for dashboards by dashboard name and by panel title. When you search for dashboards, the system returns all dashboards available within the Grafana instance, even if you do not have permission to view the contents of the dashboard.
+You can search for dashboards and folders by name.
 
-## Search dashboards using dashboard name
+When you search for dashboards, you can also do it by panel title. Whether you search by name or panel title, the system returns all dashboards available within the Grafana instance, even if you do not have permission to view the contents of the dashboard.
 
-Begin typing any part of the dashboard name in the search bar. The search returns results for any partial string match in real-time, as you type.
+## Search by name
 
-Dashboard search is:
+Begin typing any part of the dashboard or folder name in the search bar. The search returns results for any partial string match in real-time, as you type.
+
+The search is:
 
 - Real-time
 - _Not_ case sensitive
-- Functional across stored _and_ file based dashboards.
+- Functional across stored _and_ file based dashboards and folders.
 
 {{% admonition type="note" %}}
-You can use your keyboard arrow keys to navigate the results and press `Enter` to open the selected dashboard.
+You can use your keyboard arrow keys to navigate the results and press `Enter` to open the selected dashboard or folder.
 {{% /admonition %}}
 
-The following image shows the search results when you search using dashboard name.
+The following image shows the search results when you search a dashboard by name.
 
 {{< figure src="/static/img/docs/v91/dashboard-features/search-by-dashboard-name.png" width="700px" >}}
 
+The following images shows the search of a folder and the search for a dashboard inside that folder.
```

Comment:
The page starts out talking about searching for dashboards and folders by name, so maybe there should be three total examples:

- Searching by dashboard name from the Dashboards page
- Searching by folder name from the Dashboards page
- Searching by dashboard name inside a folder

Reply:
Yes. The first one, '_Searching by dashboard name from the Dashboards page_', was already on the Search page. The following two, '_Searching by folder name from the Dashboards page_' and '_Searching by dashboard name inside a folder_', are the ones that I want to include. 
Do you want me to be more specific? For example:

> The following images show:
1.- Searching by dashboard name from the Dashboards page
|>>> Here the screenshot
2.- Searching by folder name from the Dashboards page
|>>> Screenshot
3.- Searching by dashboard name inside a folder
|>>> Screenshot

Reply:
Yes, exactly! I think just so it's really clear the two different starting points and what one is searching for.